cfme/cloud/provider/__init__.py

```diff
@@ -109,6 +109,16 @@ def __init__(self, name=None, endpoints=None, zone=None, key=None, appliance=Non
         self.key = key
         self.endpoints = self._prepare_endpoints(endpoints)
 
+    def create(self, **kwargs):
```

Comment:
I added this for utility's sake in that it would automatically happen when configuring a provider. It's presence begs for a boolean arg to trigger it.

Could easily be convinced that this should be in the provider fixture instead, for testing context rather than framework context.